frontend/src/actions/dashboard/failure-count/index.spec.js

```diff
@@ -0,0 +1,70 @@
+import 'babel-polyfill'
+import Mappersmith from 'mappersmith'
+import 'mappersmith/fixtures'
+import configureMockStore from 'redux-mock-store'
+import thunk from 'redux-thunk'
+
+const middlewares = [ thunk ]
+const mockStore = configureMockStore(middlewares)
+
+import {
+  REQUEST_FAILURE_COUNT,
+  RECEIVE_FAILURE_COUNT,
+  REQUEST_FAILURE_COUNT_FAILED
+} from 'actions'
+
+import { fetchFailureCount } from 'actions/dashboard/failure-count'
+
+beforeEach(() => {
+  Mappersmith.Env.Fixture.clear()
+})
+
+describe('actions/dashboard/failure-count', () => {
+  describe('#fetchFailureCount', () => {
+    describe('when it succeeds', () => {
+      let count, store
+      beforeEach(() => {
+        count = 7
+        store = mockStore({})
+        Mappersmith.Env.Fixture
+          .define('get')
+          .matching({ url: '/api/v1/failures/count' })
+          .response(count)
+      })
+
+      it('creates REQUEST and RECEIVE actions', (done) => {
+        store.dispatch(fetchFailureCount()).then(() => {
+          const actions = store.getActions()
+          expect(actions[0]).toEqual({ type: REQUEST_FAILURE_COUNT })
+          expect(actions[1]).toEqual({ type: RECEIVE_FAILURE_COUNT, data: count })
```

Comment:
Even though the response has an object. I don't mind at all parsing it to the number in the action (not the reducer). 
So this spec, in my opinion, should stay like this still ;-)